src/Options/OptionsDetails.hpp

```diff
@@ -173,12 +174,15 @@ struct print;
 template <typename T, typename OptionList, typename = std::nullptr_t>
 struct print_impl {
   // T is a group here
-  using subgroups_t = typename options_in_group<OptionList, T>::type;
+  using filtered_options = options_in_group<OptionList, T>;
```

Comment:
With `T` -> `Group` rename it's probably clearest to use `using opts_in_group = ...` as well